packages/hardhat/contracts/FundingVault.sol

```diff
@@ -1,4 +1,4 @@
-//SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: MIT
 
 /**
```

Comment:
There is no need for boilerplate comments. Only use comments to provide important information.